BigBaseV2/src/backend/looped/weapons/no_spread.cpp

```diff
@@ -0,0 +1,51 @@
+#include "backend/looped/looped.hpp"
+#include "pointers.hpp"
+#include "CPedFactory.hpp"
```

Comment:
No need to include CPedFactory.hpp, it is already included by common.hpp.

Reply:
I'm incorrect, but still remove CPedFactory.hpp, look at one of the comments further down, you won't need the CPedFactory you can directly access the local CPed.